parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetReader.java

```diff
@@ -230,11 +230,12 @@ protected Builder(InputFile file) {
       if (file instanceof HadoopInputFile) {
         HadoopInputFile hadoopFile = (HadoopInputFile) file;
         this.conf = hadoopFile.getConfiguration();
-        this.configuration = new HadoopParquetConfiguration(this.conf);
         optionsBuilder = HadoopReadOptions.builder(this.conf, hadoopFile.getPath());
       } else {
+        this.conf = new Configuration();
         optionsBuilder = ParquetReadOptions.builder(new HadoopParquetConfiguration());
```

Comment:
The idea behind `HadoopParquetConfiguration` is to run without Hadoop. With this line we initialize a Hadoop configuration while it is not a Hadoop file.

What do you think of:
```suggestion
        this.configuration = new HadoopParquetConfiguration();
        optionsBuilder = ParquetReadOptions.builder(this.configuration);
```

Reply:
I think it's better than mine.